src/frontend/src/lib/types/errors.ts

```diff
@@ -1 +1,13 @@
+import type { TokenId } from '$lib/types/token';
+
 export class UserProfileNotFoundError extends Error {}
+
+export class AddressError extends Error {
+	tokenId: TokenId;
+
+	constructor(tokenId: TokenId) {
```

Comment:
Can it be simplied by just declaring it private?

```
export class AddressError extends Error {
   constructor(private readonly tokenId: TokenId) {
      super()
```

Reply:
yes, but then I need to create a getter to read it, right?


```typescript
	public getTokenId(): TokenId {
		return this.tokenId;
	}
```

Reply:
Ah gotcha. While a bit more verbose, it's still nice to have it as `readonly`? Maybe just adding that keyword then?

Reply:
yes, agreed! changed